.buildkite/pipeline.yml

```diff
@@ -1,4 +1,9 @@
 steps:
+  - command:
+      - GITHUB_TOKEN=$(cat ~/niv-bot-token) NIX_PATH=nixpkgs=channel:nixos-unstable nix-shell -p utillinuxMinimal jq git gitAndTools.hub niv curl --run "curl https://raw.githubusercontent.com/serokell/scratch/master/scripts/update-niv.sh | bash"
```

Comment:
Maybe something like `nix run https://serokell/scratch -c  update-niv` instead?

Reply:
Yeah, I was thinking about packaging it properly, but then I was carried away with other stuff. Will do.